### ISSUE TITLE
Users/Community Discord

### ISSUE THREAD

**Issue by Mathew3000** [rtlopez/esp-fc#92]
Hey, I noticed there is no place for the community under this project to exchange, @rtlopez would you be willing to create a Discord or some sort? I think it could be a faster solution for exchanges than GitHub issues ;)

**Comment by rtlopez** [comment]
Hi guys, I see there is interest, give me a few days ;)

**Comment by rtlopez** [comment]
here is the invitation https://discord.gg/jhyPPM5UEH, let's test ;)

**Comment by rtlopez** [comment]
I'll keep it open until I update the documentation.

**Comment by rtlopez** [comment]
@dahklfahdkghas What do you mean by 
""has expired"? this link is configured to never expire. Are you getting any error message?

**Comment by dahklfahdkghas** [comment]
Yes, the discord app says it’s invalid or expired. But if you say it never expires I’ll try updating the app or something. I never use it normally.

**Comment by bsvdoom** [comment]
> Hi all, thanks all for this great project, i wanted to start the esp-fc as a simple 4 channel rc receiver without gyro for fixed wing plane., where shall i start.

I think fixed wing is not supported yet.

If the discord server is ready, I think the invitation link can go to the README, and this ticket can be closed. Cheers
